src/test/java/com/google/cloud/discotoproto3converter/DiscoToProto3ConverterAppTest.java

```diff
@@ -246,6 +246,40 @@ public void convertAnyFieldInError() throws IOException {
     assertEquals(baselineBody, actualBody);
   }
 
+  @Test
+  public void convertWithIdempotentMerge() throws IOException {
```

Comment:
Should we remove this test since there are no assertions?  If it doesn't fail without the fixes in this PR then perhaps we could include the test in a separate PR and keep that PR open until we can add the assertion back. WDYT?

Reply:
I was debating that. I slightly lean toward keeping the test with the commented-out assertion so we have another reminder in the code that this is something we need to resolve. It succinctly captures what I'm trying to test, whereas copying into the PR without context IMO would be a bit more confusing.